dist/css/rivet-shell.css

```diff
@@ -118,6 +116,10 @@ li:first-child .rvt-list-nav__group-heading {
   color: #bbb;
 }
 
+li:first-child .rvt-list-nav__group-heading {
+  margin-top: 0;
+}
+
 .rvt-list-nav--compact li a {
   padding-top: .25rem;
```

Comment:
While you're in here working on this stuff, would you mind replacing these hard-coded values with variables?